packages/core/src/electron-main/electron-main-application.ts

```diff
@@ -292,12 +306,14 @@ export class ElectronMainApplication {
             minHeight: 120,
             webPreferences: {
                 // `global` is undefined when `true`.
-                contextIsolation: false,
+                contextIsolation: true,
+                sandbox: false,
```

Comment:
Why is this false?

----

I have noticed that you deleted a few comments pointing to GH issues; I think you can delete the `// https://github.com/eclipse-theia/theia/issues/2018` comment 👇 too as your PR is going to resolve it 😊

Reply:
sandbox is `false`, because it's not part of either the original issue nor the requirements for detachable windows. Before Electron 20, sandboxing was disabled by default. I have not investigated what that step would require, nor extensively tested with the sandbox enabled. One step at a time.